projects/unbound/parse_packet_fuzzer.c

```diff
@@ -0,0 +1,16 @@
+#include "config.h"
+#include "util/regional.h"
+#include "util/fptr_wlist.h"
+#include "sldns/sbuffer.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *buf, size_t len)
+{
+  struct regional *region = regional_create();
```

Comment:
Could you add `static` please? 
Otherwise this gets done on every input instead of just on the first one.